src/main/java/com/greenfox/dramacsoport/petclinicbackend/config/SecurityConfig.java

```diff
@@ -2,6 +2,7 @@
 
 import com.greenfox.dramacsoport.petclinicbackend.config.webtoken.JwtAuthenticationFilter;
 import com.greenfox.dramacsoport.petclinicbackend.services.MyUserDetailService;
+import lombok.RequiredArgsConstructor;
```

Comment:
Are you using this import?

Reply:
@markkovari 
No... I tried to solve it with that annotation (without Autowiring), but somehow it didn't work, so I decided not to use it and forgot to delete the import.